03-task-manager/starter/controllers/tasks.js

```diff
@@ -0,0 +1,30 @@
+const Task = require('../models/task')
+
+const getAllTasks = (req, res) => {
+    res.send('get all tasks')
+}
+
+const createTask = async (req, res) => {
+    const task = await Task.create(req.body)
+    res.status(201).json({task})
+}
+
+const getTask = (req, res) => {
+    res.send('get single Task')
+}
```

Comment:
You need to implement these methods retrieving and updating data on MongoDB. See the example located in the final/controller directory to know how to implement this.